oshdb-util/src/test/java/org/heigit/ohsome/oshdb/util/geometry/GeoTest.java

```diff
@@ -276,6 +279,161 @@ void testLengthRealFeatures() {
     assertEquals(1.0, Geo.lengthOf(line) / expectedResult, relativeDelta);
   }
 
+  @Test
+  void testRectilinearity() throws ParseException {
+    final double L = 1E-4;
+    final double D = 10;
```

Comment:
Consider adding explanations for `L` and `D` or use more understandable variables.